test/libsolidity/semanticTests/storage/storage_boundary_array_packing_not_overlapping_variable.sol

```diff
@@ -0,0 +1,60 @@
+contract C {
+    struct S {
+        uint256 value;
+    }
+
+    constructor() {
+        S storage _y = getY();
+        _y.value = 0x42424242; // Should not be overwritten
+    }
+
+    function getArray() internal pure returns (uint64[10][1] storage _x) {
+        assembly {
+            _x.slot := sub(0, 1)
+        }
+    }
+
+    function getY() internal pure returns (S storage _y) {
+        assembly {
+            _y.slot := 2
```

Comment:
This could use a comment that the array takes only three slots. I was a bit confused until I realized that it's an array of `uint64` :)